src/bloomd/networking.c

```diff
@@ -36,7 +36,13 @@
  * Default listen backlog size for
  * our TCP listener.
  */
-#define BACKLOG_SIZE 64
+#define TCP_BACKLOG_SIZE 64
+
+/**
+ * Default listen backlog size for
+ * our UNIX listener.
+ */
+#define UNIX_BACKLOG_SIZE 128
```

Comment:
minor thing, maybe just make it configurable by user? so you don't need to rebuild in case you want to change it

Reply:
Make sense. But I don't think we are going to allocate more than 100 socket connections from the client. With rolling filters and more than 100000 requests per seconds, there is some chance to have 100 concurrent requests. So I just wanted to be sure that we can handle close to 100 sequential allocations. For now, I think constant would be enough.

Reply:
it's a bit weird that the default backlog size is different from that of the tcp transport without any express justification